README.ja.md

```diff
@@ -34,23 +34,23 @@ Swayは沢山のディストリビューションで提供されています。"
 * json-c
 * pango
 * cairo
-* gdk-pixbuf2 (システムイコンで必要です)
-* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) (manで必要です) \*
-* git \*
+* gdk-pixbuf2 (任意: システムイコンで必要です)
+* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) (任意: manで必要です) \*
+* git (任意: バージョン情報で必要です) \*
 
-_\*コンパイルの時_
+_\*コンパイル時の依存_
 
 次のコマンドを実行してください:
 
     meson build
     ninja -C build
     sudo ninja -C build install
 
-logindを使用していないシステムでは、バイナリにsuidを設定する必要があります:
+logindやseatdを使用していないシステムでは、swayバイナリをsuidする必要があります:
```

Comment:
Ah. This is technically a correct README fix, but [suid support was removed some time back](https://github.com/swaywm/sway/pull/7044). We should update the English README accordingly.

Reply:
Got it. I'll remove this part.

Reply:
Done.